Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -248,5 +248,25 @@ public static void Unhover ([NotNull] this ElementScope scope)
 
       Cursor.Position = new Point (0, 0);
     }
+
+    /// <summary>
+    /// Gets an attribute from the scope or throws a <see cref="MissingHtmlException"/> if it could not be found. Wraps <see cref="ElementScope.this"/>.
+    /// </summary>
+    /// <remarks>
+    /// Accessing HTML attributes through <see cref="ElementScope.this"/> returns null if the attribute does not exist.
+    /// This might not be the desired behavior in some cases, thus offering this extension method.
+    /// </remarks>
+    /// <exception cref="MissingHtmlException">If the attribute cannot be found.</exception>
+    [NotNull]
+    public static string GetAttributeOrThrow ([NotNull] this ElementScope scope, [NotNull] string attributeName)
+    {
+      ArgumentUtility.CheckNotNull ("scope", scope);
+      ArgumentUtility.CheckNotNullOrEmpty ("attributeName", attributeName);
+
+      var result = scope[attributeName]
+                   ?? throw new MissingHtmlException ($"Cannot find the attribute '{attributeName}' on the current scope.");
+
+      return result;

```

Comment:
Inline?

Reply:
Or, actually, since the sole purpose of this method is to get the value and perform the nullcheck, should this be separate statements instead of a nullcheck tagged on to the first statement?

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/9f3d7bc46ab13a1cc811bed081e65596ccc91a3c.